.editorconfig

```diff
@@ -0,0 +1,9 @@
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = space
+insert_final_newline = true
+max_line_length = 120
+tab_width = 2
+trim_trailing_whitespace = true

```

Comment:
I guess we should also add `insert_final_newline = true`
Why? I read up some stackoverflow answers: https://stackoverflow.com/a/37684763/11829874

and ESLint doc also states this rule: https://eslint.org/docs/rules/eol-last

What do you say?

Reply:
One more reference: https://thoughtbot.com/blog/no-newline-at-end-of-file

Reply:
My bad sorry. It's already there